packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -328,7 +329,7 @@ export class MediaPlaceholder extends Component {
 		if ( mediaUpload && isAppender ) {
 			return (
 				<>
-					{ this.renderDropZone() }
+					{ ! disableDropZone && this.renderDropZone() }
```

Comment:
What are your thoughts on making this check part of the `renderDropZone` function? The thing on my mind is that if another dev adds in another drop zone to the component they have to remember to add this check before it. It might be simpler if the check were implicit.

Reply:
Sure. I don’t see a problem in that. It’s better. DRY pattern. I’ll fix that today.

Reply:
81c70b1d52a22c9a983948ca2a53bdba0f6efc2b